source/items/platearmor.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Plate Armor
+layout: default
+---
+# Plate Armor
+
+<div class="infobox box text-center">
+{% comment %}
+    <p style="text-align:center;"><img src="../../assets/images/items/platearmor.png" alt="Plate Armor"></p>
+{% endcomment %}
+</div>
+<br>
+
+Plate Armor can be crafted by the [Blacksmith](../../source/workers/blacksmith), after their [workshop](../../source/buildings/blacksmith) has reached level 4, and the Plate Armor research has been completed at the [University](../../source/buildings/university).
```

Comment:
Maybe do "Plate Armor is a type of armor added by MineColonies. It is moderately durable and offers excellent protection compared to conventional iron armor, but has poor enchantability. Plate Armor is crafted by the [Blacksmith](../../source/workers/blacksmith) after their [workshop](../../source/buildings/blacksmith) has reached level 4 and the Plate Armor [research](../../source/systems/research) has been completed at the [University](../../source/buildings/university)."